README-ruRU.md

```diff
@@ -3792,9 +3792,9 @@
 # Сотрудничество
 
 Ничто, описанное в этом руководстве, не высечено в камне. И я очень хотел бы
-сотрудничать со всеми, кто интересуется стилистикой оформления кода на Руби,
+сотрудничать со всеми, кто интересуется стилистикой оформления кода на Ruby,
 чтобы мы смогли вместе создать ресурс, который был бы полезен для всего
-сообщества программистов на Руби.
```

Comment:
Exactly! The thing is about personal preferences. Thus if we use 'Ruby' we let a reader decides on his own in which way to pronounce it :)